_format/1.1/index.md

```diff
@@ -2226,6 +2226,28 @@ because the value for the `updated` key is not recognized under the profile's
 requirement that the `updated` key hold a string of the form produced by 
 `JSON.stringify`.
 
+### <a href="#profiles-required" id="profiles-required" class="headerlink"></a> Server-Required Profiles 
+
+Generally, servers should not require that clients apply specific profiles to 
+their requests, as such requirements reduce interoperability.
```

Comment:
This is a _very_ important sentence for understanding how profiles are meant to be implemented for maximum interoperability! I'm glad we're adding it after all :)